### PR TITLE
Apply try-catch

### PR DESCRIPTION
**Pull request by theyup12**
I apply the try-catch and refactor into shorter code in lib/libOptions

### CODE REVIEW

File: lib/options.js [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-        try {
-            return {options: JSON.parse(readFile(filePath))};
-        } catch(err) {
-            return {
-                error: new Error(`Options file at ${filePath} could not be parsed with error\n${err.message}`),
-            };
-        }
+
+    const tryCatch = require('try-catch');
+    const [error, options] = tryCatch(JSON.parse, readFile(filePath));
```
Comment (coderaiser) [inline lib/options.js:10]:
```suggestion
    const [error, options] = tryCatch(JSON.parse, readFile, filePath)); 
```

File: lib/options.js [hunk 2]
```diff
@@ -1,22 +1,17 @@
...
-        }
+
+    const tryCatch = require('try-catch');
+    const [error, options] = tryCatch(JSON.parse, readFile(filePath)); 
+    if(error){
+        return {error: new Error(`Options file at ${filePath} could not be parsed with error\n${err.message}`)};
     }
-    
-    return {options: {}};
+    return options;
```
Comment (coderaiser) [inline lib/options.js:14]:
```suggestion
    return [null, options];
```

File: lib/options.js [hunk 3]
```diff
@@ -1,22 +1,17 @@
...
-        } catch(err) {
-            return {
-                error: new Error(`Options file at ${filePath} could not be parsed with error\n${err.message}`),
-            };
-        }
+
+    const tryCatch = require('try-catch');
+    const [error, options] = tryCatch(JSON.parse, readFile(filePath)); 
+    if(error){
+        return {error: new Error(`Options file at ${filePath} could not be parsed with error\n${err.message}`)};
```
Comment (coderaiser) [inline lib/options.js:12]:
```suggestion
        return [new Error(`Options file at ${filePath} could not be parsed with error\n${err.message}`), null];
```

**Review by coderaiser — CHANGES_REQUESTED** [review]
That's amazing, thank you! Please remove `ChangeLog` changes and run `npm run fix:lint`.

**Comment by theyup12** [comment]
I'm sorry, seems like a lot error, What should I do to fix it?

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,6 @@
+2021.09.17,v7.1.0
```
Comment (coderaiser) [inline ChangeLog:1]:
```suggestion
```

File: ChangeLog [hunk 5]
```diff
@@ -1,3 +1,6 @@
...
+feature:
```
Comment (coderaiser) [inline ChangeLog:2]:
```suggestion
```

File: ChangeLog [hunk 6]
```diff
@@ -1,3 +1,6 @@
...
+- (package) try-catch v3.0.0
```
Comment (coderaiser) [inline ChangeLog:3]:
```suggestion
```

**Review by coderaiser — CHANGES_REQUESTED** [review]

**Comment by coderaiser** [comment]
Run `npm run fix:lint` :).

**Comment by coderaiser** [comment]
Could you please update `bin/minif.js` and tests?

**Comment by theyup12** [comment]
I have issues on using madrun

**Comment by coderaiser** [comment]
Landed in [v7.2.0](https://github.com/coderaiser/minify/releases/tag/v7.2.0) 🎉

**Pull request closed without merging**